src/client/src/consumer.rs

```diff
@@ -391,23 +391,27 @@ impl Default for ConsumerConfig {
 }
 
 impl ConsumerConfig {
-    /// TODO describe what the max_bytes option does
+    /// Analogous to
+    /// [KIP-74](https://cwiki.apache.org/confluence/display/KAFKA/KIP-74%3A+Add+Fetch+Response+Size+Limit+in+Bytes)
```

Comment:
I found this someplace in the fluvio src as a reference but I'm not sure it's correct.